src/io/parse_ctrees.h

```diff
@@ -474,7 +478,12 @@ static inline int read_single_tree_ctrees(int fd, off_t offset, const struct ctr
                 if(*this == '\n') {
                     *this = '\0';
 
-                    assert( this >= start && this - start < PARSE_CTREES_MAXBUFSIZE);
+                    if( ! ( this >= start && (this - start) < PARSE_CTREES_MAXBUFSIZE)) {
+                        fprintf(stderr, "Error: Expected this = %p to be >= start = %p "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [%u in format string (no. 4) requires 'unsigned int' but the argument type is 'signed int'.](https://app.codacy.com/app/jacobseiler/sage-model/pullRequest?prid=4171404)